addons/account_edi_proxy_client/models/account_edi_proxy_user.py

```diff
@@ -50,18 +48,19 @@ class AccountEdiProxyClientUser(models.Model):
     edi_identification = fields.Char(required=True, help="The unique id that identifies this user for on the edi format, typically the vat")
     private_key = fields.Binary(required=True, attachment=False, groups="base.group_system", help="The key to encrypt all the user's data")
     refresh_token = fields.Char(groups="base.group_system")
+    edi_operating_mode = fields.Selection(selection=[('demo', 'Demo'), ('test', 'Test (Experimental)'), ('prod', 'Official')], string='EDI mode', required=True, readonly=True, default='demo')
 
     _sql_constraints = [
         ('unique_id_client', 'unique(id_client)', 'This id_client is already used on another user.'),
```

Comment:
I thought you would have wanted to relax the constraints with the edi_operating_mode?